pkg/provider/aws/secretsmanager/secretsmanager.go

```diff
@@ -410,19 +412,21 @@ func (sm *SecretsManager) findByTags(ctx context.Context, ref esv1beta1.External
 	var nextToken *string
 	for {
 		log.V(1).Info("aws sm findByTag", "nextToken", nextToken)
-		it, err := sm.client.ListSecrets(&awssm.ListSecretsInput{
+		it, err := sm.client.BatchGetSecretValueWithContext(ctx, &awssm.BatchGetSecretValueInput{
```

Comment:
what happens if we have a `find` operation that would target more than 20 Secrets ? 😄 Shouldn't we perform any additional logic (I would expecting at least like an `Iter` object or a `Paginator` in order to continue the search?

Reply:
But this is in a for loop already that paginates with next token.

Reply:
yup. Too soon on a saturday for accurate reviews 😄